### DISCUSSION TITLE
How to include double curly brackets in message?

### DISCUSSION THREAD

**Discussion by yogeshgalav** [intlify/vue-i18n#1660] [Q&A]
I have a string which contains double curly brackets, here is how it's look in json file
``` 
  "WELCOMEMSGHELPSTRING": "ウェルカムメッセージに組織を追加するには、{{団体名}}を使用します。例：-ウェルカム{{団体名}}",
```
This string does not contain any variables,
when using it in vue3 file
```
<span>{{ $t("WELCOMEMSGHELPSTRING") }}</span>
```
it gives me following error
```Message compilation error: Not allowed nest placeholder```
How to show this string as it is, please help

**Comment by ChrisGV04** [comment]
According to the documentation, the characters `{ } $ @ $ |` are considered special characters, so they need to be interpolated to use them as plan text. For example:

1. To write `email@company.com` you would need `email{'@'}company.com`.
2. To write `My {special} word` you would need `My {'{'}special{'}'} word`.
3. In your example, you would need to do `{'{{'}団体名{'}}'}`

I know it looks very confusing. I'm not a fan of how this plugin handles formatting compared to i18next, but that's how it must be done in this case.

Here is the documentation of what I mentioned: https://vue-i18n.intlify.dev/guide/essentials/syntax.html#literal-interpolation
